.github/workflows/docker-3.2.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         with:
           push: ${{ github.event_name != 'pull_request' }}
           context: .
-          file: Dockerfile.git
+          file: Dockerfile-alpine
```

Comment:
FWIW, a common convention I see in projects generally is a `.` followed by the variant name. The [official convention](https://docs.docker.com/build/concepts/dockerfile/#filename) suggested by Docker is the opposite to use `.Dockerfile` as the file extension.

I think `Dockerfile.` as a prefix instead of as a suffix is preferred for co-location when viewing a file list sorted by filename. It may also be recognized by editors like VSCode, I'm not sure if that's the case with `-` as a delimiter.

Regardless it doesn't differ in usage, so whatever is comfortable 😎